tests/system/Log/LoggerTest.php

```diff
@@ -345,6 +345,21 @@ public function testDebugLogsCorrectly()
         $this->assertSame($expected, $logs[0]);
     }
 
+    public function testCustomLogsCorrectly()
+    {
+        $config = new LoggerConfig();
+        $logger = new Logger($config);
+
+        $expected = 'CUSTOM - ' . date('Y-m-d') . ' --> Test message';
```

Comment:
This test depends on time. But is there no way to inject time?

Reply:
I've copied that piece of code from existing tests. There are no mocks for time there. The date is generated in Log file handler so it will have to be mocked to use injected time as well.

Reply:
I see. It depends on `date()`. We need to fix the `MockLogger` someday.